css/datagrid/datagrid.css

```
@@ -381,6 +381,10 @@
     display: block;
 }
 
+.highcharts-datagrid-table thead {
```

We usually add the `hcg-` prefix here also, in case someone for some reason uses the new prefixes in grid pro.

```suggestion
.hcg-table thead,
.highcharts-datagrid-table thead {
```